packages/commonwealth/server/controllers/server_threads_methods/update_thread.ts

```diff
@@ -325,6 +326,66 @@ export async function __updateThread(
         ],
       },
       { model: this.models.Topic, as: 'topic' },
+      {
```

Comment:
we probably have a query that can be reused here - seems like we are duplicating query logic inside a command

Reply:
There are some `include` statements that are quite similar but not exactly the same, prob because all endpoints that return a thread don't have a consistent format for the returned thread object. We could probably make a utility function to get a thread with a unified response format but this would affect a lot of areas in frontend.  

Would you say its a hard blocker? Should I make the unified response format in this PR or a follow-up if possible?

Reply:
Gotcha, thanks. I think it would be better to do it in the trpc migration phase, to not double the work